torch/_inductor/codegen/cpp_micro_gemm.py

```diff
@@ -719,9 +795,13 @@ def create_from_config(cls, config: CppMicroGemmConfig):
                 continue
             if (
                 config.input_dtype == input_dtype
-                and config.output_dtype == output_dtype
                 and config.compute_dtype == compute_dtype
                 and config.input2_dtype == input2_dtype
+                and config.output_dtype == output_dtype
+                # The output_dtype here is the output dtype of the micro-kernel.
+                # In some cases, the actual output dtype of the op for which the micro-kernel
+                # is being created would be same as that of the activation, but the micro-kernels
+                # computes output in Float, which is converted in the GEMM template.
```

Comment:
In the future, we might have some micro-gemm that produce data types other than float32. Can we revise the comment here for clarification?